.drone.yml

```diff
@@ -17,7 +17,7 @@ steps:
       INVENTORY: build:8090
       DB_HOST: postgres
       DB_PORT: 5432
-      USE_K8S: false
+      DEPLOY_TARGET: notk8s
```

Comment:
@YuviGold @oshercc we don't use drone anymore right?
If not then this file can be deleted.

Reply:
OK so things related to drone can be deleted.